help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md

```diff
@@ -0,0 +1,192 @@
+---
+keywords: email;Email;e-mail;email destinations;salesforce;salesforce destination
+title: Salesforce Marketing Cloud (ExactTarget) connection
+description: The Salesforce Marketing Cloud (ExactTarget) destination allows you to export your account data and activate it within Salesforce Marketing Cloud (ExactTarget) for your business needs.attribute set
+---
+
+# [!DNL Salesforce Marketing Cloud (ExactTarget)] connection
+
+## Overview {#overview}
+
+[[!DNL Salesforce Marketing Cloud (ExactTarget)]](https://www.salesforce.com/products/marketing-cloud/overview/) is a digital marketing suite formerly known as ExactTarget that allows you to build and customize journeys for visitors and customers to personalize their experience.
+
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [Salesforce Update Contacts REST API](https://developer.salesforce.com/docs/marketing/marketing-cloud/guide/updateContacts.html), which allows you to add contacts / update contact data for your business needs after activating them within a new Salesforce segment.
```

Comment:
Maybe we should highlight this paragraph in an IMPORTANT note. 

The reason is that we already have a Salesforce Marketing Cloud destination in the UI (https://experienceleague.adobe.com/docs/experience-platform/destinations/catalog/email-marketing/salesforce-marketing-cloud.html?lang=en). 

We need to clearly indicate the difference between the old and the new destination, namely that the old one exports files and this one is an API-based streaming connection.

Reply:
Makes sense have added the below instead of highlighting your recommendation which is more nuanced.
```
> [!IMPORTANT]
> 
> There exists a different [Salesforce Marketing Cloud connection](https://experienceleague.adobe.com/docs/experience-platform/destinations/catalog/email-marketing/salesforce-marketing-cloud.html?lang=en). However, its operation is to support file exports to a specified storage location, whereas this is an API-based streaming connection.
```